README.rst

```diff
@@ -169,14 +169,11 @@ Answer the prompts with your own desired options_. For example::
     use_heroku [n]: y
     use_compressor [n]: y
     Select postgresql_version:
-    1 - 10.5
-    2 - 10.4
-    3 - 10.3
-    4 - 10.2
-    5 - 10.1
-    6 - 9.6
-    7 - 9.5
-    8 - 9.4
+    1 - 11.3
+    2 - 10.8
+    3 - 9.6
+    4 - 9.5
+    5 - 9.4
     Choose from 1, 2, 3, 4, 5, 6, 7, 8 [1]: 1
```

Comment:
```suggestion
     Choose from 1, 2, 3, 4, 5 [1]: 1
```

Reply:
thanks for catching this. It's fixed in the newest commit.